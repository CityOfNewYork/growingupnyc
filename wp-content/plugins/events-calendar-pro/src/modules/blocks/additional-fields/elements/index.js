/**
 * Internal dependencies
 */
export { default as Settings } from './settings/container';
export { default as Preview } from './preview/element';
export { default as EditContainer } from './edit-container/element';
export { default as Field } from './field-container/container';
