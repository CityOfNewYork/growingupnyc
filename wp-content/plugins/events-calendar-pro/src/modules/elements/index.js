export { default as ExceptionAddField } from './exception-add-field/element';
export { default as ExceptionField } from './exception-field/element';
export { default as ExceptionForm } from './exception-form/element';
export { default as RecurringAddField } from './recurring-add-field/element';
export { default as RecurringField } from './recurring-field/element';
export { default as RecurringForm } from './recurring-form/element';
export { default as Panel } from './panel/element';
export { default as PanelHeader } from './panel-header/element';
export { default as Row } from './row/element';
export { default as Fieldset } from './fieldset/element';
export { default as Label } from './label/element';
export { default as LabeledRow } from './labeled-row/element';
export { default as RemoveField } from './remove-field/element';
export { default as MultiDayCheckbox } from './multi-day-checkbox/element';
export { default as SeriesEnds } from './series-ends/element';
export { default as DayOfWeekPicker } from './day-of-week-picker/element';
export { default as OnDayOfWeek } from './on-day-of-week/element';
export { default as OnDayOfMonthPicker } from './on-day-of-month-picker/element';
export { default as DayOfMonthPicker } from './day-of-month-picker/element';
export { default as RecurringToDateTimePicker } from './recurring-to-date-time-picker/element';
export { default as SingleToDateTimePicker } from './single-to-date-time-picker/element';
export { default as FromTimeRangePicker } from './from-time-range-picker/element';
export { default as MonthPicker } from './month-picker/element';
export { default as InMonth } from './in-month/element';
export { default as TypePicker } from './type-picker/element';
export { default as OnDatePicker } from './on-date-picker/element';
