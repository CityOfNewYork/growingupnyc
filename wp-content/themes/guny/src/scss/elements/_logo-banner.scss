// Logo Banner

.logo-banner {
  height: 55px;
  display: flex;
  align-items: center;
  padding: rem-calc(12);

  @include media($bp-large) {
    height: 115px;
    padding: rem-calc(24);
  }

  &--small {
    a {
      display: block;
    }
  }
}

.logo-banner__img {
  display: inline-block;
  height: 100%;
}

.logo-banner--small > .logo-banner__img {
  max-height: 55px;
  margin-top: 15px;
}

.logo-banner__link {
  margin-top: 15px;
}