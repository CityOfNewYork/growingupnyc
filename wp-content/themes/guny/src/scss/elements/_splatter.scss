// Paint Splatter
.splatter-1 {
  display: none;
  position: absolute;
  left: -50%;
  top: 0;
  opacity: .75;
  @include media($bp-medium) {
    display: block;
  }
}

.splatter-2 {
  position: absolute;
  opacity: .75;
  bottom: -50%;
  left: -90%;
  width: 200%;

  @include media($bp-large) {
    left: -50%;
    width: auto;
  }
}

.splatter-3 {
  display: none;
  position: absolute;
  opacity: .75;
  top: 0;
  right: -50%;
  @include media($bp-medium) {
    display: block;
  }
}