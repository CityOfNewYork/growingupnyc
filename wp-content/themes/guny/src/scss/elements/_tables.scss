.s-wysiwyg {
  table {
    font-size: rem-calc(16px);
    margin-bottom: rem-calc(40px);
    max-width: 100%;
    text-align: right;
    @include media($bp-medium) {
      font-size: rem-calc(18px);
    }
  }

  thead {
    border-bottom: 1px solid $very-dark-gray;
    color: $blue;

    tr {
      background-color: transparent;
    }
  }

  tbody {
    tr {
      &:nth-child(odd) {
        background-color: $light-blue;
      }
    }
  }

  th {
    vertical-align: top;
  }

  th,
  td {
    padding: rem-calc(10px);
    word-break: normal;
    @include media($bp-medium) {
      padding: rem-calc(10px 40px);
    }
  }
}
