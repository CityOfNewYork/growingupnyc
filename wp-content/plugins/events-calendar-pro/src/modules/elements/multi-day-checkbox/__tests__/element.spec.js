/**
 * External dependencies
 */
import React from 'react';
import { noop } from 'lodash';

/**
 * Internal dependencies
 */
import { MultiDayCheckbox } from '@moderntribe/events-pro/elements';

describe( 'Multi Day Checkbox Element', () => {
	it( 'renders multi day checkbox', () => {
		const component = renderer.create(
			<MultiDayCheckbox
				checked={ true }
				className="test-class"
				onChange={ noop }
			/>
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );
} );
