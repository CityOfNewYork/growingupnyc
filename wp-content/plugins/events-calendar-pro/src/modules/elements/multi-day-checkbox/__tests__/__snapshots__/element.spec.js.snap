// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Multi Day Checkbox Element renders multi day checkbox 1`] = `
<div
  className="tribe-editor__checkbox tribe-editor__multi-day-checkbox test-class"
>
  <input
    checked={true}
    className="tribe-editor__input tribe-editor__input--checkbox tribe-editor__checkbox__input"
    id="uniqid"
    onChange={[Function]}
    type="checkbox"
  />
  <label
    className="tribe-editor__checkbox__label"
    htmlFor="uniqid"
  >
    Multi-day
  </label>
</div>
`;
